source/transfer_list.rst

```diff
@@ -657,5 +658,58 @@ such that the last ACPI table in this entry ends at offset
      - hdr_size
      - One or more ACPI tables.
 
+
+.. _tpm_evlog_entry:
+
+TPM info table entry layout (XFERLIST_TPM_INFO)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This entry type holds TPM info of the platform.
+
+.. _tab_tpm_info:
+.. list-table:: TPM info type layout
+   :widths: 2 2 2 8
+
+   * - Field
+     - Size (bytes)
+     - Offset (bytes)
+     - Description
+
+   * - tag_id
+     - 0x3
+     - 0x0
+     - The tag_id field must be set to **5**.
+
+   * - hdr_size
+     - 0x1
+     - 0x3
+     - |hdr_size_desc|
+
+   * - data_size
+     - 0x4
+     - 0x4
+     - sizeof (base_address) + sizeof (evlog_size) + sizeof (crb_base_address) + sizeof (crb_size)
+
+   * - evlog_base_address
+     - 8
+     - 8
+     - The base address of event log
+
+   * - evlog_size
+     - 0x4
+     - 16
+     - Size of the event log
+
+   * - crb_base_address
```

Comment:
Why exactly do we need the CRB here?

Reply:
@apalos The CRB is a region of memory that a TPM implementation (e.g. TPM service in TrustZone) uses to communicate with TPM clients.  Firmware such as TF-A may need to carve out and reserve a region of memory for this purpose.  The "crb_base_address" field in the transfer list is so that firmware like TF-A can communicate the address/size of the carve out region to subsequent firmware components.

Reply:
Ok fair enough